libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,375 @@
+          "Name": "Triggerfish",
+          "PartyId": "1"
+        }
+      ],
+      "Contest": [
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "best-animal-fish",
+          "Name": "Best Animal",
+          "NumberElected": 1,
+          "VotesAllowed": 1,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["seahorse"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["salmon"]
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "aquarium-council-fish",
+          "Name": "Zoo Council",
+          "NumberElected": 2,
+          "VotesAllowed": 2,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["manta-ray"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["pufferfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["rockfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["triggerfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.BallotMeasureContest",
+          "@id": "fishing",
+          "Name": "Ballot Measure 3",
+          "ContestSelection": [
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "yes",
+              "Selection": "yes"
+            },
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "no",
+              "Selection": "no"
+            }
+          ],
+          "PrimaryPartyId": "1"
+        }
+      ],
+      "ElectionScopeId": ["sample-county"]
+    }
+  ],
+  "GpUnit": [
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "sample-county",
+      "Name": "Sample County",
+      "Type": "other"
+    },
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "precinct-1",
+      "Name": "Precinct 1",
+      "Type": "precinct"
+    }
+  ],
+  "CVR": [
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "ElectionId": "some-id",
+      "UniqueId": "id-168",
+      "vxBallotType": "absentee",
+      "vxTestBallot": true,
+      "CurrentSnapshotId": "id-168-original",
+      "CVRSnapshot": [
+        {
+          "@id": "id-168-original",
+          "@type": "CVR.CVRSnapshot",
+          "Type": "original",
```

Comment:
I think that the CVRs we export from our scanning devices would all be considered "original" rather than "modified." They report the selections for overvote, even if they won't count. 

Modified = "After contest rules applied."
Original = "As scanned, no contest rules applied."

It'll keep it simpler for us if we just stick to original.

Reply:
After discussing with @mattroe, I'm sure "original" is correct as we're not applying contest rules to determine selections (without straight party voting0.

Reply:
What are "contest rules"?

Reply:
Also, are we later going to include adjudication information for adjudicated ballots when exporting final CVRs from VxAdmin?

Reply:
@jonahkagan see the context around 1.1.5-F here: https://docs.google.com/document/d/15B-LjdB-i5JhcVBi6b_GxmBtke6NYiBiMhoLzisIwPA/edit

Reply:
I'm just not clear about how overvotes and undervotes factor in. I thought those could potentially fall under "contest rules". If so, I would interpret the CDF as follows:
- original: what bubbles the scanner believes the voter bubbled
- modified: the logical interpreted "vote" based on the bubbles
- adjudicated: any changes to that logical interpretation based on adjudication

So when I originally assessed this, I thought our current CVRs looked more like "modified" than "original." But my interpretation of all this could be wrong, I don't have much context

Reply:
My understanding on the categorization is the same as @jonahkagan's. Section 3.3 in the spec clarifies this pretty well too.

After reading the spec more carefully, I think the intention is that under/overvote logic is a contest/election rule after all. The CDF spec repeatedly mentions that this only applies to HMPB because DREs prevent marking outside of contest/election rules. Section 2.2 says that scanners apply interpretation regularly based on election rules, and cites overvote & undervote rules as examples of those cases.

When talking with @adghayes today, I was under the impression that we had an "original" representation in our CVR today, but I didn't double check that.

Ultimately, what I would want to see in the system is a CVR that is as auditable as possible: where the original ballot marks are in the record, any interpretation done by the scanner is specified, and the results of human adjudication are also eventually identifiable.

Reply:
@jonahkagan I've been confused about the same thing.

If our CVRs didn't include the overvoted selections, then I would think it's modified. But our CVRs do contain all the overvoted selections, so I now think of it as the "original." Although we are doing the logical interpretation for tabulation on VxScan, the CVR does not reflect that.

Reply:
I think our CVRs are "original" because, in an overvote case, we do include all of those options. Even though we do the logical interpretation for tabulation for a precinct scanner report, the CVR is pre-logical interpretation. So I think we're good here, but have been confused as well.